test/SymbolGraph/Symbols/UnavailableOnAllPlatforms.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -module-name UnavailableOnAllPlatforms -emit-module -emit-module-path %t/
+// RUN: %target-swift-symbolgraph-extract -module-name UnavailableOnAllPlatforms -I %t -pretty-print -output-dir %t
+// RUN: %FileCheck %s --input-file %t/UnavailableOnAllPlatforms.symbols.json
+
+// REQUIRES: OS=macosx
+
+// CHECK: ShouldAppear
+@available(Linux, unavailable)
```

Comment:
What happens if you have a `@available(macOS, unavailable)` attribute in this test? Since you're restricting the testing to macOS, i want to make sure that "unavailable on the platform i'm building for" still shows up in the symbol graph.

Reply:
If the symbol is marked unavailable for the platform you're currently building for, the symbol is already removed from the SGF. This is tested in the `Unavailable.swift` file.